plugin.video.BlanketFort/addon.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.BlanketFort"
+       name="BlanketFort"
+       version="1.0.0"
+       provider-name="BF Media">
+  <requires>
+    <import addon="xbmc.python" version="2.1.0"/>
+	<import addon="script.module.beautifulsoup" version="3.0.8"/>
+    <import addon="script.module.simplejson" />
+  </requires>
+  <extension point="xbmc.python.pluginsource"
+            library="default.py">
+        <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en">Official Blanket Fort Add-On</summary>
+    <description lang="en">Watch full episodes, bonus content and exclusive clips from BlanketFortLife.com</description>
+    <platform>all</platform>

```

Comment:
Pls add all the needed fields here, particularly source. Pls see http://kodi.wiki/view/addon.xml for details.